src/ApiPort.VisualStudio/ApiPort.VisualStudio.csproj

```diff
@@ -31,7 +31,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
Ahh I see! Can we move it to being enabled on release? It's easy to ignore warnings and allow them to pile up. :/